recipes/perl-dbi/build.sh

```diff
@@ -17,3 +17,9 @@ else
     echo 'Unable to find Build.PL or Makefile.PL. You need to modify build.sh.'
     exit 1
 fi
+
```

Comment:
can you remove these comments from both files please

Reply:
is this ok?